Sergey Sedelnikov/z2-1.js

```diff
@@ -0,0 +1,45 @@
+const readline = require('readline');
+const fs = require('fs');
+const minimist = require('minimist');
+const argv = minimist(process.argv.splice(2));
+
+var rl = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout
+});
+
+var randomnum;
+var file = argv._[0];
+
+console.log('Добро пожаловать в орлянку!');
+console.log('Введите 1 или 2. И пусть удача всегда будет на Вешей стороне.');
+
+rl.on('line', function (num) {
+    if((num !== '1')&&(num !== '2'))
```

Comment:
Нужно стараться поддерживать читаемость кода на высоком уровне
```suggestion
    if((num !== '1') && (num !== '2'))
```